debug.xml

```diff
@@ -65,10 +65,15 @@
     <!-- DATABASE CONFIG -->
 
     <!--<entry key='database.driverFile'>hsqldb.jar</entry>-->
-    <entry key='database.driver'>org.h2.Driver</entry>
+    <!--<entry key='database.driver'>org.h2.Driver</entry>
     <entry key='database.url'>jdbc:h2:./target/database</entry>
     <entry key='database.user'>sa</entry>
-    <entry key='database.password'></entry>
+    <entry key='database.password'></entry>-->
+
+    <entry key='database.driver'>com.mysql.jdbc.Driver</entry>
```

Comment:
Ok I will do all the changes and send new pull request

‫ב-13 ביולי 2016, בשעה 8:22, ‏‏Anton Tananaev ‏<notifications@github.com> כתב/ה:‬

> In debug.xml:
> 
> >      <entry key='database.url'>jdbc:h2:./target/database</entry>
> >      <entry key='database.user'>sa</entry>
> > -    <entry key='database.password'></entry>
> > +    <entry key='database.password'></entry>-->
> > +
> > +    <entry key='database.driver'>com.mysql.jdbc.Driver</entry>
> Yes, you need to remove it first.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>